File: fullstack/ts-short-url/src/models/shorturl.ts
import { Entity, PrimaryGeneratedColumn, Column, PrimaryColumn } from "typeorm";

// 类和成员名称要保持和数据库信息一致
@Entity("short_url")
export class short_url {
    @PrimaryColumn()
    public shorturl_id!: string;

    @Column()
    public original_url!: string;

    @Column()
    public create_date!: string;
}

File: java/src/main/java/com/example/urlshortener/model/ShortUrl.kt
package com.example.urlshortener.model

import javax.persistence.*
import javax.validation.constraints.NotNull

/**
 * @author  lilu - Li Lu (7939743@qq.com)
 * @version 1.0
 * @since   2022/04/19
 */
@Entity
@Table(name = "short_urls")
class ShortUrl {
    @Id
    @Column(name = "id")
    var id: @NotNull String? = null

    @Column(name = "original_url")
    var originalUrl: @NotNull String? = null
}